tests/execution_spec.rs

```diff
@@ -1023,6 +1024,22 @@ async fn assert_spec(spec: ExecutionSpec) {
     log::info!("[{}] {} ... ok", spec.name, spec.path.display());
 }
 
+fn start_mock_server() -> Server {
```

Comment:
why do we need additional server for tests and why mocks are not sufficient for this?

Reply:
I don't think it's good idea to put 1.4kb of data in every reflection related test.. maybe we can design some new proto file for tests but the descriptor file will always be huge

Reply:
current approach is case specific. if we add feature to specify mocks as separate files will it solve the issue?

Reply:
can you give an example?